sirepo/job.py

```diff
@@ -58,19 +51,13 @@
 #: where user data files come in (job_supervisor)
 DATA_FILE_ROOT = None
 
-#: where job_processes request files lib files for api_runSimulation
-LIB_FILE_ABS_URI = None
-
-#: where job_process will PUT data files for api_downloadDataFile
-DATA_FILE_ABS_URI = None
-
 #: how jobs request files (relative to `srdb.root`)
 SUPERVISOR_SRV_SUBDIR = 'supervisor-srv'
 
 #: how jobs request files (absolute)
 SUPERVISOR_SRV_ROOT = None
 
-DEFAULT_IP = '127.0.0.1'
+DEFAULT_HOST = socket.gethostname()
```

Comment:
we still need DEFAULT_IP, because there's no guarantee that we can lookup `gethostname` by DNS. Some routers block DNS lookups that return private IPs.